archey/entries/terminal.py

```diff
@@ -14,16 +61,22 @@ class Terminal(Entry):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
-        terminal = os.getenv(
-            'TERM',
-            self._configuration.get('default_strings')['not_detected']
+        terminal_emulator = (
+            self._detect_terminal_emulator()
+            or self._configuration.get('default_strings')['not_detected']
         )
```

Comment:
Sorry, I couldn't resist - would you mind if I opened a PR in future to change a lot of these backslash line continuations into implied continuations with brackets for consistency? I realise some of them are bracketed and some of them aren't at the moment 😕

Reply:
Please be my guest !
Wrong habit from me due to many hours on other languages too (and as PyLint does not yell at me :fearful:)...